The failure on Linux is due to:
```
/ci/myagent/_work/1/s/installed/x64-linux/debug/lib/libevent.a /ci/myagent/_work/1/s/installed/x64-linux/debug/lib/libevent_core.a /ci/myagent/_work/1/s/installed/x64-linux/debug/lib/libevent_extra.a -levent_openssl_static /ci/myagent/_work/1/s/installed/x64-linux/debug/lib/libglog.a -lpthread /ci/myagent/_work/1/s/installed/x64-linux/debug/lib/libgflags_debug.a -lpthread && :
/usr/bin/ld: cannot find -levent_openssl_static
collect2: error: ld returned 1 exit status
```
It seems that there is no event_openssl installed here.
I try to checkout this branch to my local and do as the following steps:
1.  `vcpkg install libevent[openssl] `.
2.  `vcpkg install evpp`

Then the failure disappeared.
So I guess we need to install `openssl `feature manually to test this issue.

--- a/ports/evpp/CONTROL
+++ b/ports/evpp/CONTROL
@@ -1,5 +1,5 @@
 Source: evpp
-Version: 0.7.0-1
+Version: 0.7.0-2
 Homepage: https://github.com/Qihoo360/evpp
 Description: A modern C++ network library based on libevent for developing high performance network services in TCP/UDP/HTTP protocols.
 Build-Depends: glog, libevent (windows), libevent[openssl] (!windows), rapidjson, concurrentqueue (!windows), boost-lockfree (!windows)

--- a/ports/evpp/CONTROL
+++ b/ports/evpp/CONTROL
@@ -2,4 +2,4 @@ Source: evpp
 Version: 0.7.0-1
 Homepage: https://github.com/Qihoo360/evpp
 Description: A modern C++ network library based on libevent for developing high performance network services in TCP/UDP/HTTP protocols.
-Build-Depends: glog, libevent, rapidjson, concurrentqueue (!windows), boost-lockfree (!windows)
+Build-Depends: glog, libevent (windows), libevent[openssl] (!windows), rapidjson, concurrentqueue (!windows), boost-lockfree (!windows)

